### DISCUSSION TITLE
Pop up show "The server that the application is connecting to is unverified"

### DISCUSSION THREAD

**Discussion by YenZtudio** [airsdk/Adobe-Runtime-Support#1684] [Q&A]
Pop up show "The server that the application is connecting to is unverified"
This happend when apdate AIR
I use the same config, same files but only change AIR to 31.1.1.743
(latest success [no pop-up] is version 33.1.1.633, 33.1.1.686)
please help, thank you

**Comment by fix89** [comment]
same problem, any news?

**Reply by ajwfrost** [reply]
Sorry hadn't seen this one earlier. What server is it that you're connecting to and how? i.e. is this happening within an ActionScript call using URLStream or similar?

**Reply by fix89** [reply]
Hi! I have this problem calling an URLLoader load method. On the server (www.ynfo.it) we have a "Let's Encrypt" valid certificate. With AIR 686 no problems, with new ones we received an alert
